test/ClangImporter/Inputs/many-imports/module.modulemap

```diff
@@ -0,0 +1,100 @@
+module Module1 { header "module1.h" }
```

Comment:
Mm. I added the headers in an attempt to make it fail the way it does on the actual project, but it didn't actually work. I'll take them back out.